include/chainbase/pagemap_accessor.hpp

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <fcntl.h>    // open 
+#include <unistd.h>   // pread, sysconf
+#include <cstring>
+#include <cstdlib> 
+#include <cassert>
+#include <optional>
+#include <iostream>
+#include <fstream>
+#include <filesystem>
+#include <vector>
+#include <span>
+#include <boost/interprocess/managed_external_buffer.hpp>
+#include <boost/interprocess/anonymous_shared_memory.hpp>
+
+namespace chainbase {
+
+namespace bip = boost::interprocess;
+
+class pagemap_accessor {
+public:
+   ~pagemap_accessor() {
+      _close();
+   }
+
+   bool clear_refs() const {
+      if constexpr (!_pagemap_supported)
+         return false;
+      
+      int fd = ::open("/proc/self/clear_refs", O_WRONLY);
+      if (fd < 0)
+         return false;
+      
+      const char *v = "4";
```

Comment:
`const char* v = "4";`
Would be good to add a comment why 4.
```
              4 (since Linux 3.11)
                     Clear the soft-dirty bit for  all  the  pages  associated
                     with  the  process.   This  is  used (in conjunction with
                     /proc/[pid]/pagemap) by the check-point restore system to
                     discover which pages of a process have been dirtied since
                     the file /proc/[pid]/clear_refs was written to.
```

Reply:
I added a comment.

Reply:
Thanks, fixed.